dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/importers/ui/form/ResultItem.java

```diff
@@ -0,0 +1,141 @@
+package edu.harvard.iq.dataverse.importers.ui.form;
+
+import edu.harvard.iq.dataverse.importer.metadata.ResultField;
+import edu.harvard.iq.dataverse.persistence.dataset.ControlledVocabularyValue;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResultItem implements Comparable<ResultItem> {
```

Comment:
I could use some Javadoc here, first question that comes to my mind is, result to what kind of action?